packages/lib-classifier/src/store/ClassificationStore.js

```diff
@@ -126,15 +126,24 @@ const ClassificationStore = types
         classification.metadata.update(metadata)
 
         classification.completed = true
+
         // Convert from observables
-        let classificationToSubmit = classification.toSnapshot()
+        const classificationToSubmit = classification.toSnapshot()
 
         const convertedMetadata = {}
         Object.entries(classificationToSubmit.metadata).forEach(([key, value]) => {
           convertedMetadata[snakeCase(key)] = value
         })
         classificationToSubmit.metadata = convertedMetadata
 
+        // SubjectGroup wants to submit all subject.subjectIds for analytics & retirement rules
+        const workflow = tryReference(() => getRoot(self).workflows.active)
+        if (workflow.configuration.subject_viewer === 'subjectGroup') {
```

Comment:
Since you've already got the subject, you could use `getType` here, to get the subject type directly. It'll be `SubjectGroup` for a grouped subject.

https://github.com/zooniverse/front-end-monorepo/blob/83104771c1593f8246fde0892fc2e7b5150da22e/packages/lib-classifier/src/store/SubjectStore/SubjectType/SubjectType.spec.js#L166

Reply:
```suggestion
        // SubjectGroup wants to submit all subject.subjectIds for analytics & retirement rules
        const subjectType = getType(subject)?.name
        if (subjectType === 'SubjectGroup') {
```